kore/test/Test/Kore/ASTVerifier/DefinitionVerifier/PatternVerifier.hs

```diff
@@ -638,6 +639,29 @@ test_patternVerifier =
         (DeclaredSort boolSort)
         [ asSentence boolSortSentence ]
         NeedsInternalDefinitions
+    , failureTestsForObjectPattern "Domain value - sort without DVs"
+        (ExpectedErrorMessage
+            "Sorts used with domain value must have the hasDomainValues \
```

Comment:
We should extract the text of this message to avoid duplication.

Reply:
I'm on the fence for this, using explicit text in tests has its benefits... Fixed since I'm usually biased towards liking my own code, and taking this into account tilts the balance slightly towards extracting.